core/storage/base_model/gae_models.py

```diff
@@ -644,35 +644,65 @@ def _trusted_commit(
 
                 {'cmd': 'AUTO_revert_version_number'
                  'version_number': 4}
+        """
+        VersionedModel._trusted_multi_commit(
+            [self], committer_id, commit_type, commit_message, commit_cmds)
+
+    @classmethod
+    def _trusted_multi_commit(
```

Comment:
Instead of generalizing this, I think you should implement a special-case function to handle delete-multi commits. That's because I don't think this approach generalizes conceptually to other types of commits (e.g. it would be weird to have many commit messages and commit cmds of the same type), so I would rather this not be construed as a general method that other developers may use in the future.

Ditto elsewhere. (And then you can also revert some of the private --> public method name changes, etc.)

Reply:
I'm not sure if I understand what exactly you mean here. 

Since this method is used in 4 models I think it makes sense to have it as a general method, I could change the naming of the method to something that would explicitly mention `delete_multi` or include this code inside `delete_multi` method (but then I would have some separate differently named method for `ExplorationModel` and `CollectionModel` that would handle commits for them). Then the devs wouldn't be inclined to use it.

Reply:
Just to check, this comment is resolved, right?

Reply:
Yeah, done.